docs/how-tos/production_configuration.rst

```diff
@@ -0,0 +1,245 @@
+
+
+Setting up the xAPI Pipeline
+############################
+
+**Celery**
+
+When in doubt, the simplest place to start with a production configuration is Celery tasks with batching set to 100 events or 5 seconds. You will want to add at least one additional lms-worker to handle the additional load of xAPI events and the event sink pipeline. You will also probably want to add at least one additional cms-worker to handle the new work of the course publishing event sink.
+
+**Vector**
+
+Generally the Aspects created Vector configuration should work in most cases. In Kubernetes environments you will need to make sure that a Vector pod is attached to each LMS/CMS worker.
+
+**Event bus**
+
+Similar to Celery, you should start with at least 2 event bus consumers and configure batching to 100 events or 5 seconds to start with. If you find that the event queue size is growing (see "Monitoring", below), you can add more event bus consumers and/or increase the batch size. We have tested with batch sizes up to 1000 without issue.
+
+
+Choosing ClickHouse Hosting
+###########################
+
+By default Aspects deploys a single ClickHouse Docker container as part of the Tutor install. This is not the preferred way to run a production environment! In most cases, if you can afford it, ClickHouse Cloud is the easiest and most performant way to run the service, and removes the burden of dealing with scaling, security, upgrades, backups, and other potentially difficult database management issues. Axim has been using ClickHouse Cloud for load testing and is designed to work with it.
+
+Altinity Cloud is another hosting service that Aspects has tested with in the past, but may require more hands-on integration as they use a different clustering approach than ClickHouse Cloud.
+
+Another option if you are running in Kubernetes is to use the `clickhouse-operator`_ to deploy and manage a more fault tolerant ClickHouse cluster. Aspects support for ClickHouse clusters is currently experimental, and may not support all cluster configurations without modification.
+
+
+Setting up ClickHouse
+#####################
+
+Tutor local and k8s environments should work out of the box. See Remote ClickHouse <remote-clickhouse> and ClickHouse Cluster <clickhouse-cluster> for more information on setting up hosted services.
+
+.. note::
+
+    Don't forget the usual checklist items! Make sure the server is secured, only accessible from places it needs to be, and backed up!
+
+
+Setting up Ralph
+################
+
+You can deploy `Ralph via Helm chart`_. If you are using a pipeline that involves the Ralph learning record store (Celery or an event bus), you will want to run at least two Ralph servers for fault tolerance. Generally it consumes few resources and is quite stable. If you find that response times from Ralph are high it is usually because there are too many small ClickHouse inserts and you should turn on batching or increase your batch size.
```

Comment:
Once this [PR](https://github.com/eduNEXT/tutor-contrib-pod-autoscaling/pull/7) is released, we can implement the `HPA` strategy with a single filter. The values can be used similarly to what the official helmChart recommends: https://github.com/openfun/ralph/blob/main/src/helm/ralph/values.yaml#L76-L81